docs/make.bat

```diff
@@ -0,0 +1,36 @@
+@ECHO OFF

```

Comment:
I think you do not need this file (but doesnot hurt)

Reply:
I figured it would be nice to leave it so non-technical doc authors can build the doc on windows too.